### DISCUSSION TITLE
cookie consent

### DISCUSSION THREAD

**Discussion by tired-creating-those-usernames** [uBlockOrigin/uAssets#23297] [General]
I have enabled all (after playing around with them separately) the annoyances in the filterlists, but cookie consent notice still pops up. What to do?
site - https://lastmile.lt/

**Comment by stephenhawk8054** [comment]
Fixed here: https://github.com/uBlockOrigin/uAssets/commit/3fd63103c271aa254d5785a0a903ae62cbbf863d

You can use these as custom filters while waiting for the filter lists to update:

```adb
lastmile.lt#@#+js(set-cookie, __consent, %5B%22required%22%5D)
lastmile.lt##+js(trusted-set-cookie, __consent, %5B%22required%22%5D)
```

---

cc @Yuki2718 `%5B%22required%22%5D` needs trusted scriptlets so `lastmile.lt##+js(set-cookie, __consent, %5B%22required%22%5D)` in EL Cookie won't work.

**Reply by Yuki2718** [reply]
I don't see the rule, maybe removed.

**Reply by tired-creating-those-usernames** [reply]
on laptop [brave] it seems to be ok, but on android [fennec] - no luck

**Reply by stephenhawk8054** [reply]
@tired-creating-those-usernames Can you screenshot your My filters pane in Firefox mobile?

**Reply by tired-creating-those-usernames** [reply]
![1000013655.jpg](https://github.com/uBlockOrigin/uAssets/assets/166988440/0c778ebb-b05b-4966-be01-e43ccb88204d)

**Reply by stephenhawk8054** [reply]
@tired-creating-those-usernames Tick on `Allow custom filters requiring trust`

**Reply by tired-creating-those-usernames** [reply]
🎉👍🙏
Nice!
